controllers/secretscope_controller_databricks.go

```diff
@@ -180,15 +178,6 @@ func (r *SecretScopeReconciler) submit(instance *databricksv1alpha1.SecretScope)
 	return r.Update(context.Background(), instance)
 }
 
-func (r *SecretScopeReconciler) update(instance *databricksv1alpha1.SecretScope) error {
```

Comment:
Why the update function is being deleted?

Reply:
The linter flagged that it wasn't used anywhere :-)